hecuba/IStorage.py

```diff
@@ -170,6 +173,20 @@ def _count_name_collision(self, attribute):
                                    [self._ksp])
         return len(filter(lambda (t_name, ): m.match(t_name), q))
 
+
+
+    def _build_istorage_obj(self, obj_info, so_name, storage_id):
```

Comment:
this does not create a storageobj (not only at least). Change the name. Also make it static